packages/web3-constants/evm/coingecko.json

```diff
@@ -35,6 +38,9 @@
         "Arbitrum": "",
         "Arbitrum_Rinkeby": "",
         "xDai": "xdai",
+        "Optimism": "ethereum",
```

Comment:
This field is wrong.

Reply:
It's `optimism`, is it?

Reply:
> It's `optimism`, is it?

yes but the coingecko api did not return the price of ethereum to the optimism network, I don't know now if that has changed.